File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] ans = new int[2*n];
        
        int k=0, i=0, j=n;
        
        while(j<(2*n)){
            if(k%2==0){
                ans[k++] = nums[i++];
            }
            else{
                ans[k++] = nums[j++];
            }
        }
        
        return ans;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> res;
        
        for(int ind=0; ind<n; ind++){
            res.push_back(nums[ind]);
            res.push_back(nums[ind+n]);
        }
        
        return res;
    }
};